src/routes.ts

```diff
@@ -32,6 +16,28 @@
  * GNU Affero General Public License for more details.
  *
  * You should have received a copy of the GNU Affero General Public License
- * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
+
+import type { RouteConfig } from 'vue-router'
+import ActivityAppFeed from './views/ActivityAppFeed.vue'
+import ActivityAppNavigation from './views/ActivityAppNavigation.vue'
+
+export const routes = [
+	{
+		path: '/',
+		name: 'root',
+		redirect: { path: '/all' },
+	},
```

Comment:
Do we need to change the URL of the root page? Maybe use 'all' as a fallback value in one route?

Reply:
This makes it easier as then the sidebar item for `all activities` is automatically active on the root page.
Otherwise the route does not match the navigation entry which needs more changes to work.